tests/braille-specs/tr-g2_harness.yaml

```diff
@@ -0,0 +1,312 @@
+  - ["ñ", ⠰⠈⠝]
+  - ["ò", ⠰⠈⠕]
+  - ["ó", ⠰⠈⠕]
+  - ["ô", ⠰⠈⠕]
+  - ["õ", ⠰⠈⠕]
+  - ["ö", ⠰⠪]
+  - ["ø", ⠰⠈⠕]
+  - ["ù", ⠰⠈⠥]
+  - ["ú", ⠰⠈⠥]
+  - ["û", ⠰⠈⠥]
+  - ["ü", ⠰⠳]
+  - ["ý", ⠰⠈⠽]
+  - ["þ", ⠰⠈⠞]
+  - ["ÿ", ⠰⠈⠽]
+  - ["đ", ⠰⠈⠙]
+  - ["Ē", ⠰⠠⠈⠑]
+  - ["ē", ⠰⠈⠑]
+  - ["Ė", ⠰⠠⠈⠑]
+  - ["ė", ⠰⠈⠑]
+  - ["Ę", ⠰⠠⠈⠑]
+  - ["ę", ⠰⠈⠑]
+  - ["ě", ⠰⠈⠑]
+  - ["Ģ", ⠰⠠⠈⠛]
+  - ["ģ", ⠰⠈⠛]
+  - ["Ĥ", ⠰⠠⠈⠓]
+  - ["ĥ", ⠰⠈⠓]
+  - ["Ħ", ⠰⠠⠈⠓]
+  - ["ħ", ⠰⠈⠓]
+  - ["Ĩ", ⠰⠠⠈⠊]
+  - ["ĩ", ⠰⠈⠊]
+  - ["Ī", ⠰⠠⠈⠊]
+  - ["ī", ⠰⠈⠊]
+  - ["Į", ⠰⠠⠈⠊]
+  - ["į", ⠰⠈⠊]
+  - ["Ĳ", ⠰⠠⠈⠚]
+  - ["ĳ", ⠰⠈⠚]
+  - ["Ĵ", ⠰⠠⠈⠚]
+  - ["ĵ", ⠰⠈⠚]
+  - ["Ķ", ⠰⠠⠈⠅]
+  - ["ķ", ⠰⠈⠅]
+  - ["Ĺ", ⠰⠠⠈⠇]
+  - ["ĺ", ⠰⠈⠇]
+  - ["Ļ", ⠰⠠⠈⠇]
+  - ["ļ", ⠰⠈⠇]
+  - ["Ľ", ⠰⠠⠈⠇]
+  - ["ľ", ⠰⠈⠇]
+  - ["Ŀ", ⠰⠠⠈⠇]
+  - ["ŀ", ⠰⠈⠇]
+  - ["Ł", ⠰⠠⠈⠇]
+  - ["ł", ⠰⠈⠇]
+  - ["Ń", ⠰⠠⠈⠝]
+  - ["ń", ⠰⠈⠝]
+  - ["Ņ", ⠰⠠⠈⠝]
+  - ["ņ", ⠰⠈⠝]
+  - ["Ň", ⠰⠠⠈⠝]
+  - ["ň", ⠰⠈⠝]
+  - ["Ŋ", ⠰⠠⠈⠝]
+  - ["ŋ", ⠰⠈⠝]
+  - ["Ō", ⠰⠠⠈⠕]
+  - ["ō", ⠰⠈⠕]
+  - ["Ŏ", ⠰⠠⠈⠕]
+  - ["ŏ", ⠰⠈⠕]
+  - ["Ő", ⠰⠠⠈⠕]
+  - ["ő", ⠰⠈⠕]
+  - ["œ", ⠰⠕⠑]
+  - ["Ŕ", ⠰⠠⠈⠗]
+  - ["ŕ", ⠰⠈⠗]
+  - ["Ŗ", ⠰⠠⠈⠗]
+  - ["ŗ", ⠰⠈⠗]
+  - ["Ř", ⠰⠠⠈⠗]
+  - ["ř", ⠰⠈⠗]
+  - ["Ś", ⠰⠠⠈⠎]
+  - ["ś", ⠰⠈⠎]
+  - ["Ŝ", ⠰⠠⠈⠎]
+  - ["ŝ", ⠰⠈⠎]
+  - ["Š", ⠰⠠⠈⠎]
+  - ["š", ⠰⠈⠎]
+  - ["Ș", ⠰⠠⠈⠎]
+  - ["ș", ⠰⠈⠎]
+  - ["ß", ⠰⠈⠎]
+  - ["ẞ", ⠰⠠⠈⠎]
+  - ["Ţ", ⠰⠠⠈⠞]
+  - ["ţ", ⠰⠈⠞]
+  - ["Ť", ⠰⠠⠈⠞]
+  - ["ť", ⠰⠈⠞]
+  - ["Ŧ", ⠰⠠⠈⠞]
+  - ["ŧ", ⠰⠈⠞]
+  - ["Ũ", ⠰⠠⠈⠥]
+  - ["Ƒ", ⠰⠠⠈⠋]
+  - ["Ț", ⠰⠠⠈⠞]
+  - ["Ū", ⠰⠠⠈⠥]
+  - ["Ŭ", ⠰⠠⠈⠥]
+  - ["Ů", ⠰⠠⠈⠥]
+  - ["ƒ", ⠰⠈⠋]
+  - ["Ű", ⠰⠠⠈⠥]
+  - ["Ų", ⠰⠠⠈⠥]
+  - ["Ŵ", ⠰⠠⠈⠺]
+  - ["Ŷ", ⠰⠠⠈⠽]
+  - ["Ź", ⠰⠠⠈⠵]
+  - ["Ż", ⠰⠠⠈⠵]
+  - ["Ž", ⠰⠠⠈⠵]
+  - ["Ǎ", ⠰⠠⠈⠁]
+  - ["Ẁ", ⠰⠠⠈⠺]
+  - ["Ẃ", ⠰⠠⠈⠺]
+  - ["Ẅ", ⠰⠠⠈⠺]
+  - ["Ỳ", ⠰⠠⠈⠽]
+  - ["ț", ⠰⠈⠞]
+  - ["ũ", ⠰⠈⠥]
+  - ["ū", ⠰⠈⠥]
+  - ["ŭ", ⠰⠈⠥]
+  - ["ů", ⠰⠈⠥]
+  - ["ű", ⠰⠈⠥]
+  - ["ų", ⠰⠈⠥]
+  - ["ŵ", ⠰⠈⠺]
+  - ["ŷ", ⠰⠈⠽]
+  - ["ź", ⠰⠈⠵]
+  - ["ż", ⠰⠈⠵]
+  - ["ž", ⠰⠈⠵]
+  - ["ǎ", ⠰⠈⠁]
+  - ["ẁ", ⠰⠈⠺]
+  - ["ẃ", ⠰⠈⠺]
+  - ["ẅ", ⠰⠈⠺]
+  - ["ỳ", ⠰⠈⠽]
+  - ["À", ⠰⠠⠈⠁]
+  - ["Á", ⠰⠠⠈⠁]
+  - ["Â", ⠰⠠⠈⠁]
+  - ["Ã", ⠰⠠⠈⠁]
+  - ["Ä", ⠰⠠⠈⠁]
+  - ["Å", ⠰⠠⠈⠁]
+  - ["Æ", ⠰⠠⠁⠑]
+  - ["Ç", ⠰⠠⠡]
+  - ["È", ⠰⠠⠈⠑]
+  - ["É", ⠰⠠⠈⠑]
+  - ["Ê", ⠰⠠⠈⠑]
+  - ["Ë", ⠰⠠⠈⠑]
+  - ["Ì", ⠰⠠⠈⠊]
+  - ["Í", ⠰⠠⠈⠊]
+  - ["Î", ⠰⠠⠈⠊]
+  - ["Ï", ⠰⠠⠈⠊]
+  - ["Ð", ⠰⠠⠈⠙]
+  - ["Ñ", ⠰⠠⠈⠝]
+  - ["Ò", ⠰⠠⠈⠕]
+  - ["Ó", ⠰⠠⠈⠕]
+  - ["Ô", ⠰⠠⠈⠕]
+  - ["Õ", ⠰⠠⠈⠕]
+  - ["Ö", ⠰⠠⠪]
+  - ["Ø", ⠰⠠⠈⠕]
+  - ["Ù", ⠰⠠⠈⠥]
+  - ["Ú", ⠰⠠⠈⠥]
+  - ["Û", ⠰⠠⠈⠥]
+  - ["Ü", ⠰⠠⠳]
+  - ["Ý", ⠰⠠⠈⠽]
+  - ["Þ", ⠰⠠⠈⠞]
+  - ["Ÿ", ⠰⠠⠈⠽]
+  - ["Œ", ⠰⠠⠕⠑]
+  - ["Ā", ⠰⠠⠈⠁]
+  - ["Ă", ⠰⠠⠈⠁]
+  - ["ă", ⠰⠈⠁]
+  - ["Ą", ⠰⠠⠈⠁]
+  - ["ą", ⠰⠈⠁]
+  - ["Ć", ⠰⠠⠈⠉]
+  - ["ć", ⠰⠈⠉]
+  - ["Ĉ", ⠰⠠⠈⠉]
+  - ["ĉ", ⠰⠈⠉]
+  - ["Đ", ⠰⠠⠈⠙]
+  - ["Ċ", ⠰⠠⠈⠉]
+  - ["ċ", ⠰⠈⠉]
+  - ["Č", ⠰⠠⠈⠉]
+  - ["č", ⠰⠈⠉]
+  - ["Ď", ⠰⠠⠈⠙]
+  - ["ď", ⠰⠈⠙]
+  - ["ğ", ⠰⠣]
+  - ["ş", ⠰⠩]
+  - ["Ğ", ⠰⠠⠣]
+  - ["Ş", ⠰⠠⠩]
+  - ["ı", ⠰⠔]
+  - ["I", ⠰⠠⠔]
+  - ["i", ⠰⠊]
+  - ["İ", ⠰⠠⠊]
+  - ["Punctuation simbols", ⠠⠏⠥⠝⠉⠞⠥⠁⠞⠊⠕⠝ ⠎⠊⠍⠃⠕⠇⠎]
```

Comment:
Spelling: "simbols" should by "symbols"

Reply:
Thanks Chris, again my english. :-):-)
I fixed all testcase files from the simbols word with symbols word, sorry the typos.
Now I will be doing a force-push opreation, don't surprise, I uploading all requested modifications..
I think all requested thinks from your review comments related is resolved, except en-ueb-g1.ctb related include problem with tables/tr-g1.ctb table related, I am not have ydea how can possible resolve this request without bigger risks, near by the release date.
Perhaps better keeping now end of the table the include line, and the 3.30 release development cicle we have more time and more possibility to doing more optimizations the grade 1 table related, and fixing all 168 failures if en-ueb-g1.ctb line are removed with table level.
Because we have a proofread grade 1 test file with absolute known good, this is more easyer future (and more smaller the risk a three month development cicle). :-):-)
Into the 3.30 release development cicle Oguz already suggest more fixes (for example a match opcode related fix), so absolute sure need doing changes into the 3.30 development cicle with turkish grade 1 table.
I do not know how necessary the complex operations provided by the Turkish language table in the UEB (and how much the rules are required in the en-ueb-g1.ctb UEB table in the UEB Table subtable files are finally driven). So, it is better to test this change in a multi-month development cycle based on reading real Turkish-language documents when the base 168 defective test case is solved after the Include En-ueb-g1.ctb line has been removed.
If the last part is not full good with english language, sorry, I wrote this comment part from hungarian to english translation with Google translator. :-):-)